apps/datagouvfr/priv/gettext/dataset.pot

```diff
@@ -0,0 +1,23 @@
+## This file is a PO Template file.
+##
+## `msgid`s here are often extracted from source code.
+## Add new translations manually only if they're dynamic
+## translations that can't be statically extracted.
+##
+## Run `mix gettext.extract` to bring this file up to
+## date. Leave `msgstr`s empty as changing them here as no
+## effect: edit them in PO (`.po`) files instead.
+msgid ""
+msgstr ""
+
+#, elixir-format
+msgid "A description is needed"
```

Comment:
hum I don't understand why it didn't remove those lines from the previous file (https://github.com/etalab/transport-site/blob/f83a4bf769ec761c9fc9760e22c59c0be8e21442/apps/transport/priv/gettext/en/LC_MESSAGES/dataset.po#L46) since there is only one use for this (and it's the same for the others) :thinking: